src/System.Xml.ReaderWriter/tests/Readers/CharCheckingReader/System.Xml.RW.CharCheckingReader.Tests.csproj

```diff
@@ -28,10 +28,12 @@
     <ProjectReference Include="$(CommonTestPath)\System\Xml\ModuleCore\ModuleCore.csproj">
       <Project>{3cf0cc76-4ce0-460a-ba37-657cfed39ab0}</Project>
       <Name>ModuleCore</Name>
+      <NoPackageAvailable>true</NoPackageAvailable>
```

Comment:
That's for a different change I'm working on.  Not sure how it crept into this branch... I'll remove it and look for other instances.